frontend/src/component/changeRequest/ChangeRequests/ChangeRequestFilters.tsx

```diff
@@ -1,22 +1,48 @@
-import { Box, Chip, styled } from '@mui/material';
+import { Box, Chip, styled, type ChipProps } from '@mui/material';
 import { useState, type FC } from 'react';
 
-const StyledChip = styled(Chip)(({ theme }) => ({
-    borderRadius: `${theme.shape.borderRadius}px`,
-    padding: theme.spacing(0.5),
-    fontSize: theme.typography.body2.fontSize,
-    height: 'auto',
-    '&[aria-current="true"]': {
-        backgroundColor: theme.palette.secondary.light,
-        fontWeight: 'bold',
-        borderColor: theme.palette.primary.main,
-        color: theme.palette.primary.main,
-    },
-    ':focus-visible': {
-        outline: `1px solid ${theme.palette.primary.main}`,
-        borderColor: theme.palette.primary.main,
-    },
-}));
+const StyledChip = (ariaControlTarget: string) =>
+    styled(({ ...props }: ChipProps) => (
+        <Chip variant='outlined' aria-controls={ariaControlTarget} {...props} />
+    ))<ChipProps>(({ theme, label }) => ({
+        padding: theme.spacing(0.5),
+        fontSize: theme.typography.body2.fontSize,
+        height: 'auto',
+        '&[aria-current="true"]': {
+            backgroundColor: theme.palette.secondary.light,
+            fontWeight: 'bold',
+            borderColor: theme.palette.primary.main,
+            color: theme.palette.primary.main,
+        },
+        ':focus-visible': {
+            outline: `1px solid ${theme.palette.primary.main}`,
+            borderColor: theme.palette.primary.main,
+        },
+
+        borderRadius: 0,
+        '&:first-of-type': {
+            borderTopLeftRadius: theme.shape.borderRadius,
+            borderBottomLeftRadius: theme.shape.borderRadius,
+        },
+        '&:last-of-type': {
+            borderTopRightRadius: theme.shape.borderRadius,
+            borderBottomRightRadius: theme.shape.borderRadius,
+        },
+
+        '& .MuiChip-label': {
+            position: 'relative',
+            textAlign: 'center',
+            '&::before': {
+                content: `'${label}'`,
+                fontWeight: 'bold',
+                visibility: 'hidden',
+                height: 0,
+                display: 'block',
+                overflow: 'hidden',
+                userSelect: 'none',
+            },
+        },
```

Comment:
Add an invisible, unselectable before element (which is also hidden from the a11y tree) with the label of the chip in bold, reserving space for the bold text so it doesn't increase the size of the chip. Also, center the text inside the label to avoid it drifting to the right.